spec/spec.md

```diff
@@ -82,9 +82,60 @@ A description to the reader on how the document is structured.
 
 ## Terminology
```

Comment:
I think I would appreciate a sentence saying something along the lines of "terms are defined in each specification used in this profile and this section bring them together for readability".

Reply:
done